docs/Earthfile

```diff
@@ -0,0 +1,31 @@
+# Set the Earthly version to 0.7
+VERSION 0.7
+
+# cspell: words mkdocs runable
+
+# Copy all the source we need to build the docs
+src:
+    FROM github.com/input-output-hk/catalyst-ci/earthly/docs:feat/docs3+mkdocs-material
```

Comment:
It is needed to be referenced to some release tag

Reply:
Yes but the docs in ci are not yet merged, that's why this PR can not merge yet.